webpack.config.js

```diff
@@ -110,7 +110,7 @@ module.exports = (env, argv) => ({
   devtool: argv.mode === "production" ? "source-map" : "inline-source-map",
   devServer: {
     https: createHTTPSConfig(),
-    host: process.env.HOST_IP || "0.0.0.0",
```

Comment:
this seemed wrong, in my VM environment my host IP != my VM IP, so this failed to bind.